src/chrome/content/rules/1f0.de.xml

```diff
@@ -1,17 +1,11 @@
 <!--
-	Nonfunctional subdomains:
-
-		- files		(404s)
+	Expired:
+		- xhmikosr.1f0.de
 
 -->
-<ruleset name="1f0.de (partial)" platform="cacert">
-
-	<target host="1f0.de"/>
-	<target host="www.1f0.de"/>
-
-	<!--	Cert isn't valid for !www
-		Pages redirect to !www		-->
-	<rule from="^http://(?:www\.)?1f0\.de/wp-content/"
-		to="https://www.1f0.de/wp-content/"/>
+<ruleset name="1f0.de">
+	<target host="1f0.de" />
+	<target host="files.1f0.de" />
```

Comment:
Please also add `git.1f0.de`, `www.1f0.de` and a comment for non-working `chaos.1f0.de`.